src/bits/callbacks/nexus_data_file_writer.py

```diff
@@ -48,7 +48,7 @@ def get_sample_title(self):
 nxwriter = MyNXWriter()  # create the callback instance
 """The NeXus file writer object."""
 
-if "NEXUS_DATA_FILES" in iconfig:
+if iconfig.get("NEXUS_DATA_FILES", {}).get("IS_ON", False):
```

Comment:
```suggestion
if iconfig.get("NEXUS_DATA_FILES", {}).get("enable", False):
```

matching suggestion in iconfig.yml